### ISSUE TITLE
Exception when secure desktop opens

### ISSUE THREAD

**Issue by Danstiv** [mltony/nvda-console-toolkit#8]
Hi!

Steps to reproduce:
1. Press alt+ctrl+del.
2. Press escape.
3. See the NVDA log.

In my case i see the following
```
  File "C:\Users\Danstiv\AppData\Roaming\nvda\addons\consoleToolkit\globalPlugins\consoleToolkit.py", line 1294, in chooseNVDAObjectOverlayClasses
    if getConfig("controlVInConsole") and obj.windowClassName == 'ConsoleWindowClass':
                                          ^^^^^^^^^^^^^^^^^^^
AttributeError: '_SecureDesktopNVDAObject' object has no attribute 'windowClassName'
```

NVDA version: alpha-33464,df6bd077 (2024.4.0.33464)
Addon version: 1.4 (stable).

**Comment by cary-rowen** [comment]
I'll provide a fix PR to resolve this issue.

**Comment by cary-rowen** [comment]
@mltony 
Would you mind releasing a new version with this fix?

**Comment by mltony** [comment]
Just released v1.4.1 - available in add-on store!
